codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -906,6 +916,21 @@ public DefaultStream createStream(int streamId) throws Http2Exception {
         }
 
         @Override
+        public DefaultStream createIdleStream(int streamId) throws Http2Exception {
+            return createStream(streamId, IDLE);
+        }
+
+        @Override
+        public DefaultStream createStream(int streamId, boolean halfClosed) throws Http2Exception {
```

Comment:
Why are we changing this? I thought we had previously abandoned combining creation with opening to simplify the interface.